app/routes/synonyms.$synonymId.tsx

```diff
@@ -61,81 +98,103 @@ export async function action({
 }
 
 export default function () {
-  const { eventIds } = useLoaderData<typeof loader>()
+  const { eventIds, options } = useLoaderData<typeof loader>()
+  const uniqueOptions = Array.from(new Set(options)) as string[]
   const [deleteSynonyms, setDeleteSynonyms] = useState([] as string[])
   const [synonyms, setSynonyms] = useState(eventIds || [])
   const [addSynonyms, setAddSynonyms] = useState([] as string[])
-  const [newSynonym, setNewSynonym] = useState('')
+  const uniqueSynonyms = Array.from(new Set(synonyms)) as string[]
```

Comment:
Under what circumstances would the synonyms fail to be unique? I think our data model guarantees that they will be unique.

Reply:
Example: If the user inputs GRB 230919A and then using the input again searches and selects 230919A, only 1 230919A is added to the synonym group ul. This prevents showing multiples of the same event id like this:
<img width="327" alt="Screenshot 2024-10-10 at 10 14 03 AM" src="https://github.com/user-attachments/assets/3014e151-09b7-4a33-9fb0-45c197b5ae6f">

With the uniqueness check, it only displays one of each synonym even when added multiple times:
<img width="259" alt="Screenshot 2024-10-10 at 10 15 44 AM" src="https://github.com/user-attachments/assets/97edc5ce-a91a-40cc-a116-2cb78f62eeb1">

Either way, it will only be sent to the server once, so it won't break anything. It could just be confusing if someone added it twice and expected there to be more list items upon refresh than it has since it will remove the duplications.